modules/rest/build.gradle

```diff
@@ -59,22 +56,33 @@ dependencies {
     testImplementation project(':ignite-configuration')
     testImplementation(testFixtures(project(':ignite-core')))
     testImplementation(testFixtures(project(':ignite-configuration')))
+    testImplementation(testFixtures(project(':ignite-security')))
     testImplementation libs.slf4j.jdk14
+    testImplementation libs.micronaut.junit5
+    testImplementation libs.micronaut.http.client
     testImplementation libs.mockito.core
     testImplementation libs.hamcrest.core
     testImplementation libs.hamcrest.optional
+
+    integrationTestAnnotationProcessor libs.micronaut.inject.annotation.processor
+    integrationTestAnnotationProcessor testFixtures(project(':ignite-core'))
+    integrationTestAnnotationProcessor testFixtures(project(':ignite-cluster-management'))
+
+    integrationTestImplementation project(':ignite-rest-api')
+    integrationTestImplementation project(':ignite-network')
+    integrationTestImplementation project(':ignite-api')
+    integrationTestImplementation project(':ignite-security')
+    integrationTestImplementation testFixtures(project(':ignite-core'))
+    integrationTestImplementation testFixtures(project(':ignite-cluster-management'))
+    integrationTestImplementation testFixtures(project(':ignite-configuration'))
+    integrationTestImplementation libs.micronaut.junit5
+    integrationTestImplementation libs.micronaut.test
+    integrationTestImplementation libs.micronaut.http.client
+    integrationTestImplementation libs.micronaut.http.server.netty
+    integrationTestImplementation libs.typesafe.config
 }
 
 compileJava {
     options.fork = true
     options.forkOptions.jvmArgs += '-Dmicronaut.openapi.config.file=openapi/openapi.properties'
 }
-
-shadowJar {
```

Comment:
Should we still leave a TODO to a ticket about shadowing reactor-core later?

Reply:
Not sure that we actually need it. Keep discussing.

Reply:
Have decided to shadow reactor in rest module, because scalecube uses another version.